src/components/HotGiftRankingGrid.tsx

```diff
@@ -0,0 +1,126 @@
+import { generateMockArray } from "@/__mock__/generate-mock-array";
+import styled from "@emotion/styled";
+import { useState } from "react";
+
+const GridContainer = styled.div(props => ({
+  display: "grid",
+  gridTemplateColumns: "repeat(3, 1fr)",
+  gap: `${props.theme.spacing2}`,
+  rowGap: `${props.theme.spacing6}`,
+}));
+
+const GridItem = styled.div(props => ({
```

Comment:
```suggestion
const GridItem = styled.div({theme}=> ({
```

아래 코드는 props를 잘 구조분해할당(Destructuring Assignment) 하셨던데..
여기도 적용해주시면 조금 더 일관성 있는 코드가 될 거 같습니다!